scripts/py_to_ino_RGB_LED_test.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+
+# Importing Libraries
+# For Arduino Communication:
+# https://create.arduino.cc/projecthub/ansh2919/serial-communication-between-python-and-arduino-e7cce0
+
+# import serial
+# Refer to pyserial docs: https://pyserial.readthedocs.io/en/latest/pyserial.html
+# python3 -m pip install pyserial
+# import time
+import numpy as np
+import rospy
+
+
+# ROS Data Types
+
+# Custom Script
+import image_inputs as input_image
+import light_painting.msg #custom message type
+import RGBState
+
+
+# arduino = serial.Serial(port='/dev/ttyACM0',baudrate=9600,timeout=0.1)
+# Code from: https://www.electronicshub.org/controlling-arduino-led-python/
+
+def RGB(red,green,blue):
+
+    arduino.write(red)
+    arduino.write(green)
+    arduino.write(blue)
+    print('RGB should be on')
+
+
+def RGB_values(rgb_img):    
+    ''' Testing reading values from RGB
+
+    # print('Size of RGB: ',rgb_img.size)
+    # print('rgb_img[0,0] pixel 1 value:',rgb_img[0,0])
+    # print('rgb_img.item(0) pixel 1 value:',rgb_img.item(0)) # gets only the 1st value
+    # print('rgb_img[0,0] pixel 4 value(mid pixel):',rgb_img[1,1])
+    # print('rgb_img[0,0] pixel 8 value(bottom right):',rgb_img[2,2])
+    # red,green,blue = rgb_img[1,1]
+    # print('Red value of pixel 4:',red)
+    # print('Green value of Pixel 4:',green)
+    # print('Blue value of pixel 4:',blue) 

```

Comment:
Please remove this block of print statements. Not used by the actual code.
Not helpful for debugging. 

Suggest adding a sentence or two explaining what this function does, you already kind of have it with the top comment "Testing reading values....". Maybe just add what the input value needs to be and what the function returns.

Reply:
A comment about the specific order of RGB vs BGR may be good to include since you mention it a few lines below as well.